.github/workflows/dependabot-fix.yml

```diff
@@ -33,14 +33,13 @@ jobs:
           token: ${{ secrets.LITCHBLICK_GITHUB_TOKEN }}
           ref: ${{ github.event.pull_request.head.ref }}
 
-      - run: git lfs pull --include .yarn/
-
-      - run: corepack enable
-
       - uses: actions/setup-node@v4.0.2
         with:
           node-version: 16.17
-          cache: yarn
```

Comment:
I still need to understand the reason why this should be removed to make it work.

Reply:
https://github.com/actions/setup-node

Check if it is really needed or only an optional optimisation

Reply:
Actually what I understood was that since we are using corepack we can't configure yarn as a cache provider. Corepack is the package manager in this case and not yarn